expression/bench_test.go

```diff
@@ -1079,11 +1078,36 @@ func genVecEvalBool(numCols int, colTypes []types.EvalType) (CNFExprs, *chunk.Ch
 	return exprs, input
 }
 
+func generateRandomSel() []int {
```

Comment:
use rand.Seed to initialize the random seed?

Reply:
Update. PTAL